src/ixdat/exporters/ecms_exporter.py

```diff
@@ -1,36 +1,67 @@
-from .csv_exporter import CSVExporter
 from .ec_exporter import ECExporter
+from ..tools import deprecate
 
 
-class ECMSExporter(CSVExporter):
+class ECMSExporter(ECExporter):
     """A CSVExporter that by default exports potential, current, selector, and all MID"""
 
     @property
     def default_export_columns(self):
         """The default v_list for ECExporter is V_str, J_str, and sel_str"""
         v_list = (
-            ECExporter(measurement=self.measurement).default_v_list
+            ECExporter(measurement=self.measurement).default_export_columns
             + self.measurement.mass_list
         )
 
         return v_list
 
+    @deprecate("0.2.0", "use `columns` instead", "0.3.0", kwarg_name="v_list")
     def export(
         self,
         path_to_file=None,
         measurement=None,
         v_list=None,
+        columns=None,
         tspan=None,
         mass_list=None,
         mol_list=None,
+        time_step=None,
     ):
-        if not v_list:
+        """Export a given measurement to a specified file.
+
+        To improve flexibility with inheritance, this method allocates its work to:
```

Comment:
Good, but, as I understand it, that probably means that these methods fall through to `CSVExporter` via `ECExporter`, which is now what it inherits from. That is great, and all as intended, but I'm not quite sure that I think the comment will help a less programming inclined reader will understand that from the comment, but I may just be reading it wrong. What about something like:
```text
This method delegates the majority of the export work, via inheritance, to:
 - ...
 - ...
 - ...
```

Reply:
In any case, I would suggest "delegates" instead of "allocates" in your original comment